tools/remote_build/rbe_common.bazelrc

```diff
@@ -70,7 +70,7 @@ build:msan --crosstool_top=@com_github_bazelbuild_bazeltoolchains//configs/ubunt
 build:tsan --copt=-gmlt
 # TODO(jtattermusch): use more reasonable test timeout
 build:tsan --test_timeout=3600
-build:tsan --test_tag_filters=-qps_json_driver,-json_run_localhost
+build:tsan --test_tag_filters=-json_run_localhost
```

Comment:
this is re-enabling qps_json_driver, json_run_localhost not re-enable. Probably not intended.